app/utils/database.py

```diff
@@ -970,7 +970,7 @@ def can_edit(self, identifier):
         `identifier` will be parsed as:
 
         `^([a-zA-Z][0-9]+-[0-9]+)$`: individual
-        `^([a-zA-Z][0-9]+)$`: herd
+        `^(([G|M]X1)|[a-zA-Z][0-9]+)$`: herd
```

Comment:
`[G|M]` should be `[GM]` or just `.`.

Reply:
If we merge the first PR I can rebase on Develop

Reply:
The first one is merged. Go ahead and rebase and merge this one too.